zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -748,6 +749,16 @@ private void removeAllAngularObjectInParagraph(String user, String paragraphId)
   }
 
   public void persist(AuthenticationInfo subject) throws IOException {
```

Comment:
`forcePersist` is for `run` related calls that would be persisted always. persist on the other hand have added condition check if `persist on run` enabled or not, and it's called by other non `run` related calls